lefthook.yml

```
@@ -0,0 +1,18 @@
+    lint:
+      glob: "*.{js,ts,tsx}"
+      run: bun run lint
+    
+    typecheck:
+      glob: "*.{ts,tsx}"
+      run: bun run typecheck
+
+commit-msg:
+  commands:
+    commitlint:
+      run: bunx commitlint --edit {1}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)